src/comm/tcp_server.cpp

```diff
@@ -26,7 +26,6 @@
 //----------------------------------------------------------------------
 
 #include <ur_client_library/log.h>
-#include <ur_client_library/comm/tcp_server.h>
```

Comment:
This include is needed in order for the library to build.

Reply:
Looks like I removed the wrong include...

Reply:
Fixed in 6d010f37c81befbe84330b8a940cf3a2a278dda8 and ed39496ad723dbb98d6f3d525408c86b72a774d8

I will cleanup the branch later...